docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+gi # Welcome to Tkinter-Designer contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project!
+
+Read our [Code of Conduct](../CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](../README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+## Getting started
+
+If you find something that can be improved, solved or changed you are welcome to do so.
+
+### Issues
```

Comment:
Is there suppose to be text here under `Issues`?

Reply:
Nope, I left it by mistake. I'll remove it.